docs/MAINTAINERS.adoc

```diff
@@ -374,6 +374,9 @@ LTS backporting, if needed, will be handled separately by the release team.
   Security hardening and enhancements go through the standard process.
 * Release Team members are permitted to bypass the review/merge process if and only if a change is needed to unblock the security release.
   Common review process is used otherwise.
+* Only one approval is required for low-risk small changes with the `skip-changelog` label,
+  as long as the approver has write access to the repository.
```

Comment:
```suggestion
  as long as both author and approver have write access to the repository.
```

?

Reply:
I'm fine with tightening this constraint, since in practice I haven't seen any changes piling up from outside contributors. The one exception I can think of is minor Dependabot changes (e.g. updates to test dependencies) which I have seen pile up a few times. Ideally I would like to invoke this exception in cases like that. Technically the robot has write access to the repository, but to clarify your intent: would that seem alright to you?

Reply:
Probably depends on the bump. Assuming we have dependabot only doing trivial stuff and exclude the rest, that would be fine.

Reply:
Yeah, there's inherently some degree of subjectivity in things like this. Sometimes even two reviews and 24 hours are not enough (e.g., for Groovy version bumps). But given the number of active reviewers and the fact that many of the original authors of the code have moved on, I think this new rule strikes a reasonable compromise.